### ISSUE TITLE
Ensure we're getting stream clones

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#1405]
Stream clones should be quite fast, we should make sure we're using them when cloning the Mercurial repository.

**Comment by marco-c** [comment]
Logs while cloning:
```
2020-04-04T16:15:00.867290+00:00 app[worker.1]: 2020-04-04 16:15:00,867:INFO:bugbug_http.boot:Cloning autoland in /tmp/bugbug-hg...
2020-04-04T16:15:03.173322+00:00 app[worker.1]: (using Mercurial 5.3)
2020-04-04T16:15:31.797062+00:00 app[worker.1]: 
2020-04-04T16:15:36.793881+00:00 app[worker.1]: 
2020-04-04T16:15:38.548407+00:00 app[worker.1]: 
2020-04-04T16:16:14.955909+00:00 app[worker.1]: 
2020-04-04T16:18:28.673486+00:00 app[worker.1]: 
2020-04-04T16:18:28.673486+00:00 app[worker.1]: 
2020-04-04T16:18:28.673503+00:00 app[worker.1]: (sharing from new pooled repository 8ba995b74e18334ab3707f27e9eb8f4e37ba3d29)
2020-04-04T16:18:28.673504+00:00 app[worker.1]: applying clone bundle from https://s3-external-1.amazonaws.com/moz-hg-bundles-us-east-1/mozilla-central/cf48e59e88dec9bd29a53e2ed1a836537a439426.stream-v2.hg
2020-04-04T16:18:28.673505+00:00 app[worker.1]: 570502 files to transfer, 2.81 GB of data
2020-04-04T16:18:28.673506+00:00 app[worker.1]: transferred 2.81 GB in 188.7 seconds (15.2 MB/sec)
2020-04-04T16:18:28.673507+00:00 app[worker.1]: finished applying clone bundle
2020-04-04T16:18:28.673508+00:00 app[worker.1]: searching for changes
2020-04-04T16:18:28.673508+00:00 app[worker.1]: adding changesets
2020-04-04T16:18:28.673508+00:00 app[worker.1]: adding manifests
2020-04-04T16:18:28.673509+00:00 app[worker.1]: adding file changes
2020-04-04T16:18:37.371549+00:00 app[worker.1]: received pushlog entry for unknown changeset c98c89a11df28e970c975088f1cfefc847aa06eb; ignoring
2020-04-04T16:21:29.360162+00:00 app[worker.1]: added 49999 pushes
2020-04-04T16:21:29.360175+00:00 app[worker.1]: added 164 changesets with 614 changes to 394 files
2020-04-04T16:21:29.360176+00:00 app[worker.1]: new changesets 16ee5dd004d3:2941034e5f82
2020-04-04T16:21:29.360176+00:00 app[worker.1]: searching for changes
2020-04-04T16:21:29.360176+00:00 app[worker.1]: no changes found
2020-04-04T16:21:29.360183+00:00 app[worker.1]: received pushlog entry for unknown changeset c98c89a11df28e970c975088f1cfefc847aa06eb; ignoring
2020-04-04T16:21:50.685384+00:00 app[worker.1]: added 49999 pushes
2020-04-04T16:21:50.685412+00:00 app[worker.1]: (pulling to obtain tip)
2020-04-04T16:21:50.685421+00:00 app[worker.1]: (remote resolved tip to 2941034e5f82ce0ed135ff929775bf85f145017c; result is not deterministic)
2020-04-04T16:21:50.706239+00:00 app[worker.1]: (revision already present locally; not pulling)
2020-04-04T16:21:54.985368+00:00 app[worker.1]: (warning: large working directory being used without fsmonitor enabled; enable fsmonitor to improve performance; see "hg help -e fsmonitor")
2020-04-04T16:25:01.758080+00:00 app[worker.1]: 
2020-04-04T16:25:01.758080+00:00 app[worker.1]: 
2020-04-04T16:25:01.758093+00:00 app[worker.1]: updated to 2941034e5f82ce0ed135ff929775bf85f145017c
2020-04-04T16:25:02.785648+00:00 app[worker.1]: 2020-04-04 16:25:02,756:INFO:bugbug.repository:/tmp/bugbug-hg cloned
```

**Comment by marco-c** [comment]
I've asked @cgsheeh, we are getting stream clones.
